tests/test_verbs.py

```diff
@@ -69,9 +69,7 @@ def test_identify(app):
         "and non-commercial service providers",
         "URL": "http://arXiv.org/arXiv_metadata_use.htm",
     }
-    dataPolicy = {
-        "text": "Full content, i.e. preprints may" "not be harvested by robots"
-    }
+    dataPolicy = {"text": "Full content, i.e. preprints maynot be harvested by robots"}
```

Comment:
```suggestion
    dataPolicy = {"text": "Full content, i.e. preprints may not be harvested by robots"}
```

Reply:
Funilly it was `maynot` before (because of string concatenation), but as you mentioned, the value is never tested explicitly.